### PR TITLE
README: Add OmegaPoker Goerli and (New) Kovan Addresses + Minor Changes

### PR DESCRIPTION
**Pull request by naszam**
- Add OmegaPoker Goerli Address.
- Add New OmegaPoker Kovan Address (refreshed and poked).
- Fix typos, errors.
- Improve Readability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,9 @@ MegaPoker curent Mainnet Address: [0x1cFd93A4864bEC32C12c77594c2ec79DeeC16038](h
 
 # OmegaPoker
 
-Extensible poker for Kovan and backup for Mainnet.
+Extensible poker for Goerli, Kovan and backup for Mainnet.
+
+OmegaPoker Goerli Address: [0x091971A1DdDF12450D0D7003A29cD9fb94f8e54c](https://goerli.etherscan.io/address/0x091971a1dddf12450d0d7003a29cd9fb94f8e54c#code)
 
 OmegaPoker Kovan Address: [0x7e2531587d25b9fc58aA051D0f19c0Ae809316E9](https://kovan.etherscan.io/address/0x7e2531587d25b9fc58aa051d0f19c0ae809316e9#code)
```
Comment (brianmcmichael) [inline README.md:16]: The goerli update looks good, but I noticed that the Kovan OmegaPoker is on an old version. We can either remove Kovan or deploy a new version and update the address.

Comment (naszam) [reply README.md:18]: @gbalabasquer ?

Comment (brianmcmichael) [reply README.md:18]: It looks like the one linked here hasn't been poked in a few months. Techops I think has the new address for Kovan but I seem to have lost it in the migration from RocketChat. Let me see if I can track it down from their keeper address.

Comment (brianmcmichael) [reply README.md:18]: This is the most recent one on Kovan I think: https://kovan.etherscan.io/address/0xaf6d20dbca12bcc79ac787d606a7ff77670666f8#code

It didn't get updated after the transition, we should deploy a new one on Kovan and let techops know.

Comment (brianmcmichael) [reply README.md:18]: Just deployed:

https://kovan.etherscan.io/address/0xB73c7D82956C9DFd17F8D212b9Cc9062ac04799e#code

**Review by brianmcmichael — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (naszam) [reply README.md]: Done in f9fa45c and pinged @dizzy in chat.

Comment (naszam) [reply README.md]: Kovan OmegaPoker is Live:
https://kovan.etherscan.io/tx/0x0bedbd74336f6a393733e8159e050108b2b4ad31d19971de118ddbb90272171d

**Pull request merged**